canonical/store-components
# Link card icon

## Done
Made the card into icon a link

## QA
- Go to https://store-components-40.demos.haus/?path=/story/bundlecard--default-story
- Check that the card icon is a link
